TestCases/tutorials.py

```diff
@@ -56,6 +57,18 @@ def main():
     cht_incompressible_unsteady.tol       = 0.00001
     test_list.append(cht_incompressible_unsteady)
 
+    # CHT incompressible, 2D, 3 pins in crossflow
+    cht_incompressible           = TestCase('cht_incompressible')
+    cht_incompressible.cfg_dir   = "../Tutorials/multiphysics/steady_cht"
+    cht_incompressible.cfg_file  = "cht_2d_3cylinders.cfg"
+    cht_incompressible.test_iter = 10
```

Comment:
The steady_cht case that is in the tutorials is equal to the coupled_cht/incomp_2d. steady_cht was not in the regression tests which I think is desirable for all tutorials.

My proposal for this would be:
- Make sure the same reg-case works in tutorials and Testcases
- change some cfg-values (e.g. density) of the tutorial case back to its original to be the same as in the written tutorial. Make sure the results are reasonable. (actually the results may have been created with the values from the Testcases as they were to be changed [here in the related tutorial PR](https://github.com/su2code/su2code.github.io/pull/26))
- Remove the Testcases reg-test as we dont need it twice imo

It might also makes sense to move the disc_adj_incomp_2d to tutorials as that is also done in the [tutorial](https://su2code.github.io/tutorials/Inc_Heated_Cylinders/). This will/would be done in the related [Tutorials PR](https://github.com/su2code/Tutorials/pull/16)

Reply:
In case someone knows how to make the CI-pipeline clone a specific branch of su2code/tutorials ... that would be appreciated.

Reply:
In regression.yml change what is in from of "-t":
```
with:
          args: -b ${{github.ref}} -t develop -c develop -s ${{matrix.testscript}}
```
Based on this: https://github.com/su2code/Docker-Builds/blob/master/test/runTests.sh

Reply:
Yep, I am stupid. I looked at this very line and thought "this is probably affects both: Testcases and Tutorials. But I cant change one without the other" ... that why there are two params 🤦
thanks